index.html

```diff
@@ -0,0 +1,153 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>First NSS Quiz </title>
+  <link rel="stylesheet" type="text/css" href="quiz.css">
+</head>
+<body>
+  <div class="container">
+    <article>
+      <header class="title">
+        <h1>Targaryen</h1>
+      </header>
+          <!-- First section has product image, descrip, avail. Text justified  -->
+          <section>
+            <img src="http://www.hbo.com/assets/images/series/game-of-thrones/downloads/targaryen/wallpaper-targaryen-sigil-1600.jpg">
```

Comment:
So this totally works and is fine. But technically the requirement asks that 'The first section should contain three child block elements.' and 5-i asks 'The first element contains the product image.'
You don't have to change it if you don't want, but it would be great practice to put your images in a block level element and then try to recenter them